openapi/data_repository_service.swagger.yaml

```diff
@@ -194,57 +194,71 @@ definitions:
           sha512
   Bundle:
     type: object
-    required: ['id', 'object_ids', 'created', 'updated', 'version', 'checksums']
     properties:
       id:
         type: string
-        description: |-
+        description: >-
           An identifier, unique to this Data Bundle
       object_ids:
         type: array
         items:
           type: string
-        description: |-
+        description: >-
           The list of Data Objects that this Data Bundle contains.
       created:
         type: string
         format: date-time
-        description: |-
+        description: >-
           Timestamp of object creation in RFC3339.
       updated:
         type: string
         format: date-time
-        description: |-
-          Timestamp of update in RFC3339, identical to create timestamp in systems
-          that do not support updates.
+        description: >-
+          Timestamp of update in RFC3339, identical to create timestamp in
+          systems that do not support updates.
       version:
         type: string
-        description: |-
-          A string representing a version, some systems may use checksum, a RFC3339
-          timestamp, or incrementing version number. For systems that do not support
-          versioning please use your update timestamp as your version.
+        description: >-
+          A string representing a version, some systems may use checksum, a
+          RFC3339 timestamp, or incrementing version number. For systems that do not
+          support versioning please use your update timestamp as your version.
+      mime-type:
```

Comment:
I don't think  `mime-type` makes sense on bundles -- aren't they always just a collection of other objects? (The referenced objects can of course be typed, but that's different.)

Reply:
If you are able to access the data in a bundle via an `access_url` then mime-type may be important. It also allows future extensions for container formats that can be accessed as a bundle and or individually.

Reply:
Removed Bundle `mime-type` from this PR. Will place it in a different PR for discussion.